shared/src/vyxal/Globals.scala

```diff
@@ -25,6 +25,13 @@ case class Globals(
     String,
     CustomDefinition,
   ] = Map()
+  var classes: Map[
+    String,
+    CustomClass,
+  ] = Map()
```

Comment:
This formatting looks off. Did scalafmt split this up onto multiple lines? If it's broken, we might want to file a bug report

Reply:
copilot probably